components/brave_rewards/resources/shared/components/newtab/rewards_card.style.ts

```diff
@@ -197,12 +197,27 @@ export const balanceInfo = styled.div`
   margin-top: 18px;
 
   .tooltip {
+    visibility: hidden;
+    transition: visibility 0s linear 300ms;
+  }
+
+  .tooltip-arrow:before {
```

Comment:
Curious if you need the `:before` here, or if the `.tooltip-arrow` element can itself just be styled like this.

Reply:
Yup, we don't need a pseudo-element in this case — nice! 👌🏻